imports/ui/components/dashboard/dashboard.jsx

```diff
@@ -3,11 +3,14 @@ import DashboardSidebar from './dashboard_sidebar'
 import MatchesSection from './matches_section'
 import DashboardCardsSection from './dashboard_cards_section'
 
-const sectionTargets = {
-  'mentors' : 'mentees',
-  'mentees' : 'mentors', 
-  'oss-projects': 'oss-contributors',
-  'oss-contributors': 'oss-projects'
+const sectionTargets = { //see: categories.js for numbers
+  'all': 'all',
+  'mentors' : 3,
```

Comment:
I'm not quite sure I understood why we need to identify the sections with number IDs from checking out the categories.js file and this commit https://github.com/codebuddies/cb-connect/commit/ffb488e8e3b669cf29faf90b9524f6de2842bba7.

Did it have something to do with using them as IDs to pass to `withTracker()` for Mongo? 

If we really do have a good reason to use numerical values for this, let's at least add additional notes for our reasoning.

Reply:
They needed to be numbers because in our query to the database, we were looking for something like:

```
Entries.find({userId: {$ne: currentUser._id}, "category.id": "3"}).fetch()```

(Replace "3" with another number to indicate another category.)

Reply:
Ah! Makes more sense now, thanks! Would be helpful to add that as a comment here. 

This is interesting -- it's a bit confusing here as in the, it looks like `category.id` is defined as a String based on the Entry schema:
https://github.com/codebuddies/cb-connect/blob/8611a9a4338a7795801f833250f97452f317c766/imports/api/entries/entries.js#L9-L10

Considering we're most likely not going to have _a lot_ of other categories, does it make more sense to just the category title as the id? (not important in this PR now but perhaps a separate discussion down the line).

Reply:
The problem with using `category.title` as the ID is that the `title` might change if we decide to rename any of the short labels in `lib/data/categories.js`. IMO it's safer to refer to the categories by their numbers, since for example one day we might want to rename `short_label: 'Contributor(s)',` to be `short_label: 'Open Source Contributors'. 

Along the same lines, in the most recent batch of commits, I removed references to a `section` array and imported  `lib/data/categories.js`, so that we weren't rewriting the section labels but referring back to the source.

Reply:
I didn't actually know about `categories.js` until this branch - perhaps if we're trying to make sure that we can keep our categories always in synched, we should just import the categories from that lib file so that whenever we update it, it will always be updated everywhere else. Thoughts?

Reply:
I think that would be ideal. :) I did it in dashboard.jsx with ```    const sections = categories.map((section) => {
        return section.short_label;
    })``` but two other places I left because the refactoring would have taken a bit more work, and I was thinking we'd come back to it later. At the moment, if I update the short_label strings in categories.js, it will break the other other parts of the files that reference those string names directly, so part of the refactoring === changing the references to be the category numbers, instead of the strings.